app/src/main/java/com/sobolevkir/playlistmaker/Utils.kt

```diff
@@ -1,22 +1,22 @@
 package com.sobolevkir.playlistmaker
 
 import android.content.Context
-import android.util.TypedValue
+import android.view.inputmethod.InputMethodManager
+import androidx.appcompat.app.AppCompatActivity
 import java.text.SimpleDateFormat
 import java.util.Locale
 
 object Utils {
 
-    internal fun dpToPx(dp: Float, context: Context): Int {
-        return TypedValue.applyDimension(
-            TypedValue.COMPLEX_UNIT_DIP,
-            dp,
-            context.resources.displayMetrics
-        ).toInt()
-    }
-
     internal fun formatTrackTime(trackTimeMillis: Int): String {
         return SimpleDateFormat("mm:ss", Locale.getDefault()).format(trackTimeMillis)
     }
 
+    internal fun closeKeyboard(activity: AppCompatActivity, context: Context) {
```

Comment:
Вместо объекта Utils и функций лучше делать extension функции. Например эту функцию можно сделать как экстеншн к активити

Reply:
обе этих функции можно сделать как расширение?

Reply:
и как это расширение сделать?

Reply:
и где его прописывать?

Reply:
```suggestion
    fun AppCompatActivity.closeKeyboard() {
```
Писать такие экстеншены можно в отдельном файле например `KeyboardExtensions` или `ActivityExtensions`.